lua-cassandra-1.2.3-0.rockspec

```diff
@@ -10,7 +10,9 @@ description = {
   license = "MIT"
 }
 dependencies = {
-  "luabitop"
+  "luabitop",
+  "luasec",
+  "luasocket"
```

Comment:
Those dependencies are deliberately not included in this rockspec because they are not needed unless the user runs this library in a non-cosocket context (non-OpenResty or an OpenResty context that doesn't support cosocokets)

Reply:
Should the be listed in DEV_ROCKS then? They are needed for tests and they don't get installed by anything right now

Reply:
DEV_ROCKS sounds good! :+1: